README_es.MD

```diff
@@ -0,0 +1,287 @@
+
+`cfn-lint docs AWS::Lambda::Function.Code.S3Bucket`
+
+**Nota: El orden de `--parameters` y` --pseudo` actualmente importa. Esto debería abordarse en una nueva versión.**
+
+[//]: # (Translation Section: Example Output)
+### Ejemplo de salida
+```
+0 infos
+0 warn
+2 crit
+Recurso: Recursos> MyInstance> Propiedades
+Mensaje: Requiere propiedad de imagenId faltante por tipear. AWS::EC2::Instance
+Documentación: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance.html
+
+Recurso: Recursos> MyInstance> Propiedades
+Mensaje: Falta el valor de propiedad requerido para el tipo de etiqueta
+Documentación: http://docs.aws.amazon.com/search/doc-search.html?searchPath=documentation-guide&searchQuery=Tag&x=0&y=0&this_doc_product=AWS+CloudFormation&this_doc_guide=User+Guide&doc_locale=en_us#facet_doc_product=AWS%20CloudFormation&facet_doc_guide=User%20Guide
+
+¡Plantilla inválida!
+```
+[//]: # (Translation Section: Flags)
+### Flags
+`--parameters <param values>`: Proporcione una lista de Comas separadas llave=valor <tt>key=value</tt> por pares de parámetros para utilizar al validar su plantilla. Si un parámetro no se especifica aquí, `cfn-lint` adivinará un valor falso basado en el tipo de parámetro y valores permitidos. e.g.
+  - `--parameters InstanceType=t2.micro,Memory=512`
+
+`--pseudo <psuedo param values>`: Proporcione una lista de comas separadas llave=valor <tt>key=value</tt> por pares de CloudFormation pseudo-parámetros para utilizar al validar su plantilla. p.ej.
+  - `--pseudo AWS::Region=ap-southeast-2`
+
+`--guess-parameters`:Adivina los parámetros si no tienen ningún valor predeterminado en la plantilla. Los parámetros serán adivinados/estimados en base a sus `AllowedValues` o` Type`. Este es el comportamiento estándar; solo está incluido como una opción de claridad.
+
+`--no-guess-parameters`:Desactive la adivinación de parámetros si no tienen un valor predeterminado. Si no los proporciona en la CLI en esta situación, se generará un error crítico en lugar de estimar el valor del parámetro.
+
+`--only-guess-parameters <param names>`: Solo adivine los parámetros proporcionados, y deshabilitar la adivinación de todos los demás sin valores predeterminados. Se generará un error crítico para los parámetros faltantes, como se indicó anteriormente. p.ej.
+ - `--only-guess-parameters InstanceType,Memory`
+
+[//]: # (Translation Section:What can cfn-lint do?)
+### ¿Qué puede hacer cfn-lint?
+* <tt>Read JSON + YAML (Including YAML short form)</tt> Lea JSON + YAML (Incluyendo YAML en su formato más pequeño)
+* <tt>Detect invalid property names</tt> Detecta nombres de propiedad inválido
+* <tt>Detect invalid Ref</tt> Detecta Ref inválido
+* <tt>Detect invalid mappings</tt> Detecta mapeos inválidas
+* <tt>Detect invalid format</tt> Detecta formato inválido
+* <tt>Missing or invalid AWSTemplateFormatVersion</tt> Faltante o inválido Versión de formato de plantilla AWS
+* <tt>Fn::FindInMap</tt> (Encontrar en mapa)
```

Comment:
Can these be like the above, without the brackets ()?

ex.
`* <tt>Fn::FindInMap</tt> Encontrar en mapa`